package.json

```diff
@@ -12,7 +12,8 @@
     "start:dev": "babel-watch index.js",
     "build": "rm -rf dist && babel ./ -d ./dist --copy-files --ignore ./node_modules",
     "postinstall": "npm run build",
-    "test": "npm run db:undo-migration && npm run db-migrate && nyc --reporter=lcov --reporter=text mocha test/index.js --timeout=20000 --require babel-register --exit"
+    "test": "npm run db:undo-migration && npm run db-migrate && nyc --reporter=lcov --reporter=text mocha test/index.js --timeout=20000 --require babel-register --exit",
+    "localTest": "export NODE_ENV=localTest && npm run db:undo-migration && npm run db-migrate && nyc --reporter=lcov --reporter=text mocha test/index.js --timeout=20000 --require babel-register --exit"
```

Comment:
I think this wouldn't work because every time we want to run this localTest command, we have to make a change to the config.js file. Let's stick with the way it was before or change the `DATABASE_URL` in the config.js to be an environmental variable.

Reply:
Ok. Thanks, I will revert the changes.